sshmux.go

```diff
@@ -65,12 +66,14 @@ type AuthResponse struct {
 	PrivateKey string `json:"private_key"`
 	Cert       string `json:"cert"`
 	Id         int    `json:"vmid"`
+	Proxy      *bool  `json:"proxy_protocol,omitempty"`
```

Comment:
I really hate this... But it is the only way to handle optionals in Go without 3rd party niceties.